soft_bfddpd/events.c

```diff
@@ -48,6 +47,15 @@ struct pollfd_ctx {
 	void *pfc_arg;
 };
 
+enum timer_state_flag {
+	/** Mark as removed from timers tree. */
+	TSF_REMOVED = (1 << 0),
```

Comment:
Prefix/

Reply:
Sorry I did not understand this comment.

Reply:
Also prefix with `SBFD_` or s/t alike

Reply:
It is already prefixed with `TSF_` and it is not exported (only used internally)... I would recommend people to use battle tested event libraries instead of this (e.g. `libevent`, `libuv` etc...). The motivation was mostly to avoid external dependencies.

I even made some effort to avoid external data structures libraries, but I couldn't escape some.